session/session.go

```diff
@@ -13,43 +13,35 @@ import (
 // Create attempts to create a session using the node types provided
 // until the first successful creation. If none of the node types are successful, it
 // returns 503 out of capacity error.
-func Create(ctx context.Context, params types.ExecCreateParams, nodeTypeIDs []string) (string, error) {
```

Comment:
These are different gpuTypes to try as backup. Eg. the list could be `[a100, rtx_4000, rtx_5000]`. In this case, the CLI tries creating an `a100`, and if there's no capacity, it tries the `rtx_4000` and the `rtx_5000` in that order. Removing this breaks this behavior.

I might be that we don't want this behavior which is a different issue.

Reply:
refactored to meet expected functionality